### PR TITLE
Add API to delete consumer group by id

### PR DESCRIPTION
**Pull request by IldarAlmakaev**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [x] Manually(please, describe, when necessary)
- [x] Unit checks
- [x] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [ ] My changes generate no new warnings(e.g. Sonar is happy)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

File: kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml [hunk 1]
```diff
@@ -433,6 +433,34 @@ paths:
...
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ConsumerGroupIds'
+
```
Comment (germanosin) [inline kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:452]: Please remove empty line

Comment (germanosin) [reply kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml:452]: May be should we add endpoint: delete /api/clusters/{clusterName}/consumerGroups/{id} instead?

**Review by germanosin — CHANGES_REQUESTED** [review]

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ClusterService.java [hunk 2]
```diff
@@ -273,4 +284,47 @@ private KafkaCluster updateCluster(String topicToDelete, String clusterName,
...
+        .collect(Collectors.toList());
+  }
+
+  @NotNull
+  @SneakyThrows
+  private ConsumerGroupDeleteResult getConsumerGroupDeleteResult(
+      Map.Entry<String, KafkaFuture<Void>> groupIdAndFuture) {
+    try {
+      KafkaFuture<Void> future = groupIdAndFuture.getValue();
+      Void result = future.get(1, TimeUnit.SECONDS);
```
Comment (DementevNikita) [inline kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ClusterService.java:317]: Please remove blocking operation -- use ClusterUtils#toMono

Comment (IldarAlmakaev) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ClusterService.java]: Yes, thank you. Fixed!

**Review by germanosin — APPROVED** [review]

**Pull request merged**